dev/archery/archery/crossbow/reports.py

```diff
@@ -284,7 +284,7 @@ class CommentReport(Report):
         'github': _markdown_badge.format(
             title='Github Actions',
             badge=(
-                'https://github.com/{repo}/workflows/Crossbow/'
+                'https://github.com/{repo}/actions/workflows/crossbow.yml/'
                 'badge.svg?branch={branch}'
```

Comment:
I am not entirely sure this will work as when I update this:
https://github.com/ursacomputing/crossbow/workflows/Crossbow/badge.svg?branch=actions-635103f8ad-github-wheel-macos-mojave-cp312-amd64
to this:
https://github.com/ursacomputing/crossbow/workflows/crossbow.yml/badge.svg?branch=actions-635103f8ad-github-wheel-macos-mojave-cp312-amd64
it fails to load the badge.

Reply:
oops, I see @kou just merged :)
there is an issue opened for it btw: https://github.com/apache/arrow/issues/37927
but again, I am not sure this solves the problem

Reply:
ok, it works if I use the correct URL: https://github.com/ursacomputing/crossbow/actions/workflows/crossbow.yml/badge.svg?branch=actions-635103f8ad-github-wheel-macos-mojave-cp312-amd64, my bad

Reply:
Oh, sorry!